pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -1113,6 +1114,26 @@ private static void TeamCityFinishTestSuite(CommandLineArgs commandLineArgs)
             bool clientMode = parallelMode == "client" || parallelMode == "server_worker";
             bool asNightly = offscreen && qualityMode;  // While it is possible to run quality off screen from the Quality tab, this is what we use to distinguish for treatment of perf tests
 
+            // If running Nightly tests, remove any flagged for exclusion by the NoNightlyTesting custom attribute
+            if (asNightly)
+            {
+                for (var t = testList.Count; t-- > 0;)

```

Comment:
Could be done with two fairly simple Linq expressions which I prefer to 14 lines of in-place for loops.

Reply:
Nice use of the "[goes to 0 operator](https://stackoverflow.com/questions/1642028/what-is-the-operator-in-c)" though! `--> 0`

Reply:
Yeah, I just copied existing code which also shows my C++ roots. I can tighten that up, sure.